test/runnable/testaa2.d

```diff
@@ -62,9 +62,15 @@ void foo2()
     value = c.values;
     assert(value.length == 2);
 
-    for (i = 0; i < key.length; i++)
+    // Make items always print in the same order so the test runner can validate
+    // the output despite the order of c.keys being unspecified.
+    foreach (isItTimeForB; 0 .. 2)
     {
-        printf("c[\"%.*s\"] = %d\n", cast(int)key[i].length, key[i].ptr, value[i]);
+        for (i = 0; i < key.length; i++)
+        {
+            if ((key[i][0] == 'b') == isItTimeForB)
+                printf("c[\"%.*s\"] = %d\n", cast(int)key[i].length, key[i].ptr, value[i]);
+        }
```

Comment:
Please remove this output and add appropriate `assert`s (the test is compiled with enabled assertions).

Reply:
Done

Reply:
Thanks